test/infrastructure/docker/templates/clusterclass-in-memory.yaml

```diff
@@ -126,6 +126,13 @@ spec:
           - type: Ready
             status: "False"
             timeoutSeconds: 300
+        unhealthyMachineConditions:
+          - type: "Ready"
```

Comment:
Let's use NodeReady in this file for unhealthyMachineConditions (same below)

Currently the CC is not valid (anymore, since the CEL change)

Same for clusterclass-quick-start.yaml